x/feegrant/internal/types/errors.go

```diff
@@ -0,0 +1,24 @@
+package types
+
+import (
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+// Codes for governance errors
+const (
+	DefaultCodespace = ModuleName
+)
+
+var (
+	// ErrFeeLimitExceeded error if there are not enough allowance to cover the fees
+	ErrFeeLimitExceeded = sdkerrors.Register(DefaultCodespace, 1, "fee limit exceeded")
```

Comment:
This is just a question. Is this the defacto way of custom/module-level error implementation? In the evidence module I simply use the old paradigm of a function wrapper to return `sdkerrors.New(...)`.

Reply:
This should be the de-facto reference I believe: https://github.com/cosmos/cosmos-sdk/blob/master/types/errors/errors.go#L16

Register ensures there are never two errors registered with the same codespace and code